Dockerfile.redhat

```diff
@@ -260,7 +260,7 @@ RUN if [ "$NVIDIA" == "1" ] ; then true ; else exit 0 ; fi ; \
 
 # Build OpenVINO Model Server
 WORKDIR /ovms
-COPY .bazelrc .bazelversion WORKSPACE /ovms/
+COPY .bazelrc .user.bazelr[c] .bazelversion WORKSPACE /ovms/
```

Comment:
Why [c] ?

Reply:
It won't trigger error if file doesn't exist.